openshift/deploy/job/cas-shelf-tfe-add-app.yml

```
@@ -0,0 +1,35 @@
+
```

```suggestion
apiVersion: template.openshift.io/v1
kind: Template
```
This file needs to be an openshift template